apps/notepad/notepad.py

```diff
@@ -0,0 +1,22 @@
+from talon import Context, Module, actions
+
+ctx = Context()
+mod = Module()
+
+mod.apps.notepad = r"""
+os: windows
+and app.exe: notepad.exe
+"""
+
+ctx.matches = r"""
+app: notepad
+"""
+
+
+@ctx.action_class("win")
+class win_actions:
+    def filename():
+        filename = actions.win.title().split(" - ")[0]
+        if "." in filename:
```

Comment:
Part of of PR1451 https://github.com/talonhub/community/pull/1451 is to allow extra filename matching where a . isn't included, for example allowing the filename Dockerfile to trigger docker tags. This won't work with that type of file.

Reply:
That pull request doesn't seem to care that all other applications use this implementation already

https://github.com/talonhub/community/blob/b337c2193d30ae67be2a10b3dd4c5487f1c6821c/apps/vscode/vscode.py#L150-L151
https://github.com/talonhub/community/blob/b337c2193d30ae67be2a10b3dd4c5487f1c6821c/apps/notepad++/notepad++_win.py#L70-L72
https://github.com/talonhub/community/blob/b337c2193d30ae67be2a10b3dd4c5487f1c6821c/apps/visualstudio/visual_studio.py#L107-L109
etc..

Reply:
Yep, is why I didn't actually request a change here, just saying.

Reply:
Ok since this appears to be the community standard we just leave it for now and have to think about a solution in the future on how to support filenames without a dot in them. Sounds good?